app/assets/stylesheets/moderators.scss

```diff
@@ -213,36 +213,205 @@
     margin: 90px auto;
     border-radius: 12px;
   }
+
+  // Reaction Buttons START
+
+  .thumb-reactions {
+    display: flex;
+    justify-content: center;
+  }
+
   .reaction-button {
     position: relative;
-    width: 80px;
-    height: 80px;
     background: white;
     border-color: white;
-    margin: 30px 30px;
-    border-radius: 200px;
-    border: 0px;
-    padding: 12px 12px;
+    border: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 156px;
+    height: $su-10;
+    padding: $su-4 $su-6;
+    margin: $su-4 $su-2 $su-4 $su-4;
+    border-radius: $br-default;
+
+    &:last-child {
+      margin: $su-4 $su-4 $su-4 $su-2;
+    }
+
+    &:hover {
+      background: rgba(59, 73, 223, 0.05);
+    }
     &:active {
-      border: 0px;
+      border: 0;
+    }
+    &:hover .reaction-button-circle {
+      border-color: var(--accent-brand);
     }
   }
 
-  .reaction-button img {
+  .reaction-button-circle {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 0 auto;
+    width: $su-9;
+    height: $su-9;
+    border: solid 2px var(--base-20);
+    border-radius: 100%;
+    background: $white;
+  }
+
+  .reaction-button-circle img {
+    width: $su-7;
+  }
+
+  img.upvote {
```

Comment:
Rather than manipulate another image to use the "upvote" emoji, I can export one for you so we can use that.

Reply:
Sounds good Lisa! If it's exported by tomorrow (Thurs Apr 23 EST) then I'll use it. Thanks!